### ISSUE TITLE
Take snapshot not working as expected

### ISSUE THREAD

**Issue by hplos** [jordenc/com.synology.surveillance#17]
Hi,
When my garage-door is opened, I get a notification (whithout image).
Now I have a foscam installed and pared with my synology. I would love to get a image of my garage when the door is opened, but this only works 1 time.

I receive the image as a notification, but after that the spinner keeps spinning in the 'take photo as token' my flow.

This is strange, because the app 'knows't that the photo is taken, since I've got another flow that is activated when a photo is taken.

**Comment by hplos** [comment]
after restarting your app, I can take one photo again :)

**Comment by rowins** [comment]
Same problem here

**Comment by treecamp** [comment]
Got the same here

**Comment by larsnet** [comment]
Have the same issue, i can take one photo and then need to restart the app. 
I use the Snapshot to image token card.

**Comment by hplos** [comment]
Yes, you can restart the app after sending the push notification, but I ended up using the foscam app for my camera, this one is also faster in taking a snapshot.

Good luck!

**Comment by larsnet** [comment]
any update regarding this problem?

**Comment by KRiZ-R** [comment]
Same here, the spinner keeps spinning in the flow.

**Comment by nilsaw11** [comment]
Hello, same problem here.... only **one** picture is possible.
after **restart** the app it is posible to take a picture again.

i have found a workaround  for nou till this is soloved.

**enable "power user" in experimental settings**
nou you can make a card thats can **restart** the app.
you have to make some delays (find your correct numbers, with will work for u) 

I make this sequense 

hope you can used your way :-)

--------------------------------------------------
IF
1. this flow is started

AND
empty

THEN
2 -  make picture as token
3 - [DELAY 5 SEC] send picture to phone (or what els you want)
4. - [DELAY 15 SEC] restart surveillance app

**Comment by Shadowz7** [comment]
Same here, 
any update regarding this problem?

**Comment by hplos** [comment]
Try to use another app :) my foscam is supported directly, this al also faster that when my NAS is in the middle

Now my NAS and homey communicate directly to my cam

**Comment by larsnet** [comment]
why close? I don't have a foscam , and my smartwares camera's don't have a webinterface with the newer firmware. So i don't have a alternative at the moment :(

**Comment by treecamp** [comment]
Yeah please reopen @hplos. Other people, are still experiencing this issue, including myself.

**Comment by hplos** [comment]
Ok, than I have to find out a way to no longer receive notifications of this thread

**Comment by larsnet** [comment]
I found a other workaround , this PHP script https://github.com/sjauquet/YAPUSS
Place it in the web (webstation) folder of your Synology and modify the SSS_Get.ini with your credentials.
In the SSS_Get.php i modified the following line:
$SessionFile = substr(basename($_SERVER['SCRIPT_NAME']).PHP_EOL, 0, -4)."session";
to:
$SessionFile = "/volume1/web/SSS_session/SSS_Get.session";
SSS_session is a subfolder with write permissions to everyone (for storing the session key).

Run SSS_Get.php from your browser to see all the capabilities.
To get a snapshot: http://xxx.xxx.xxx.xxx/SSS_Get.php?stream=jpeg&camera=1&snapQual=2
Make a virtual camera in Homey and place the link there.
